### DISCUSSION TITLE
Bulk clone records

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5761] [Feature Requests]
> When entering collection objects from newly collected material, it would be convenient if the collection object could be cloned multiple times.
> For example, if I want to insert 32 identical objects, it would be nice to have a “clone it X times” option where I could set X to 32 to generate 32 records.
> It would probably be inappropriate to have this option for all collections, but is it possible to make it optional on the collection object form, or even put it in a different subform?

**Requested By:** [@cicadina on the Speciforum](https://discourse.specifysoftware.org/t/multiple-records-cloning/1110)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/multiple-records-cloning/1110/2

**Comment by grantfitzsimmons** [comment]
Somewhat related to #545
